src/components/ItemSummary.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from 'react';
+import { ItemData } from "../LibraryUtilities";
+import { LibraryContainer } from "./LibraryContainer";
+
+interface ItemSummaryProps {
+    libraryContainer: LibraryContainer,
+    data: ItemData;
+    showDescription: boolean;
+}
+
+interface ItemSummaryStates {
+    hasItemSummaryData: boolean;
+}
+
+export class ItemSummary extends React.Component<ItemSummaryProps, ItemSummaryStates> {
+    itemSummaryData: any;
+
+    constructor(props: ItemSummaryProps) {
+        super(props);
+        this.state = ({ hasItemSummaryData: false });
+        this.itemSummaryData = null;
+        this.onReceiveDataFromDynamo = this.onReceiveDataFromDynamo.bind(this);
```

Comment:
Oops, the `librarie.js` has nothing to do with Dynamo (which is just one of its consumers), so a better name would be `onDataReceived` or `handleReceivedData`.